client/txn_test.go

```diff
@@ -75,7 +75,7 @@ func TestTxnRequestTxnTimestamp(t *testing.T) {
 		{makeTS(20, 1), makeTS(20, 1)},
 	}
 
-	testIdx := 0
+	var testIdx int
```

Comment:
sort of. It complains that the explicit zero is never used because the value is overwritten before being read